src/store/modules/position.store.js

```diff
@@ -223,24 +253,16 @@ const actions = {
     },
     zoomToExtent: ({ commit, getters, rootState }, extent) => {
         if (extent && Array.isArray(extent) && extent.length === 2) {
-            // Convert extent points to WGS84 as adding the coordinates in EPSG:3857 gives incorrect results.
-            const points = [
-                proj4(WEBMERCATOR.epsg, WGS84.epsg, extent[0]),
-                proj4(WEBMERCATOR.epsg, WGS84.epsg, extent[1]),
-            ]
-            // Calculate center of extent and convert position back to EPSG:3857.
-            // Based on: https://github.com/Turfjs/turf/blob/v6.5.0/packages/turf-center/index.ts
-            const centerOfExtent = proj4(WGS84.epsg, WEBMERCATOR.epsg, [
```

Comment:
What about this comment ? Why this hack is not needed anymore ?

Reply:
I've reverted this portion and adapted the code to take `state.projection` into account